.github/workflows/run-tests.yml

```diff
@@ -47,4 +47,4 @@ jobs:
         run: poetry install --no-interaction
 
       - name: Test with unittest
-        run: poetry run python -m unittest
+        run: poetry run python -m unittest discover -s tests
```

Comment:
Does the README need a similar update?

The tests pass for me if I run `poetry run python -m unittest discover -s tests` as it's written here, but if I follow the README instructions, I get a failure:

```
judoscale-python $ poetry shell  
Spawning shell within /Users/adam/Library/Caches/pypoetry/virtualenvs/judoscale-python-gpoIQ17w-py3.10
judoscale-python $ emulate bash -c '. /Users/adam/Library/Caches/pypoetry/virtualenvs/judoscale-python-gpoIQ17w-py3.10/bin/activate'

(judoscale-python-py3.10) judoscale-python $ python -m unittest                                                                                               
E....
======================================================================
ERROR: judoscale.flask (unittest.loader._FailedTest)
----------------------------------------------------------------------
ImportError: Failed to import test module: judoscale.flask
Traceback (most recent call last):
  File "/Users/adam/.asdf/installs/python/3.10.9/lib/python3.10/unittest/loader.py", line 470, in _find_test_path
    package = self._get_module_from_name(name)
  File "/Users/adam/.asdf/installs/python/3.10.9/lib/python3.10/unittest/loader.py", line 377, in _get_module_from_name
    __import__(name)
  File "/Users/adam/src/judoscale-python/judoscale/flask/__init__.py", line 1, in <module>
    from judoscale.flask.judoscale import Judoscale
  File "/Users/adam/src/judoscale-python/judoscale/flask/judoscale.py", line 1, in <module>
    from flask import request
ModuleNotFoundError: No module named 'flask'


----------------------------------------------------------------------
Ran 5 tests in 0.013s

FAILED (errors=1)
```

Reply:
Ah yes, that's because `python -m unittest` is also picking up the sample app tests, but since the library and the sample apps have different virtualenvs, it cant' find "module named 'flask'".

The `-s tests` option scopes it to the `tests/` directory.

I think going forward we're going to use the "extras" feature in Python's packaging functionality, so we can actually convert this repo to use a single virtualenv which has all the deps for the sample apps as well.